site/content/internal/sustained-engineering.md

```diff
@@ -83,7 +83,15 @@ During non-release weeks, triage is held twice a week. Near the end of the relea
 
 ## Support Responses
 
-Part of SET's responsibility is to interface with the customer support team. SET's primary goal in respect to support is to triage escalated support issues, work on any high priority customer bugs requiring code change, as well as provide training and knowledge share with the customer support team. To help us do this better, SET members should use the following responses when receiving issue escalations from the customer support team:
+Part of SET's responsibility is to interface with the customer support team. SET's primary goal in respect to support is to triage escalated support issues, work on any high priority customer bugs requiring code change, as well as provide training and knowledge share with the customer support team. When customer support team requires enginerring support, they should:
+
+1. Posts the issue on [Customer Support](https://community.mattermost.com/private-core/channels/community) / [SET](https://community.mattermost.com/core/channels/sustained-engineering) channel
+2. Engage someone from SET to provide an initial assement
+3. If no prompt solution is presented, a ticket should be created by customer support and assigned to SET for further investigation
+
+This process helps increase accountability and tracibility for both teams.
```

Comment:
What do you mean by "both teams"?

Reply:
Engineering and customer support. Maybe I should remove `for both teams?`

Reply:
Got it, perhaps something like:

```suggestion
This process increases accountability and traceability across Mattermost.
```

Reply:
Sorry, I missed this note @ali-farooq0. How about

```suggestion
This process helps increase accountability and traceability across teams.
```

Reply:
ah good catch, missed that too. Updated.